docs/protocol.html

```diff
@@ -114,6 +114,32 @@
 
 <p>Currently all versions are baselined at 0, as we evolve these APIs we will indicate the format for each version individually.</p>
 
+<h5><a id="api_versions" href="#api_versions">Retrieving Supported API versions</a></h5>
+<p>In order for a client to successfully talk to a broker, it must use requests versions' supported by the broker. Clients may work against
+    multiple broker versions, however to do so the clients need to know what versions of various APIs does a broker supports. Clients may
```

Comment:
versions of various APIs does a broker supports -> versions of various APIs a broker supports